_health-care/_js/components/financial-assessment/Child.jsx

```diff
@@ -29,7 +29,7 @@ class Child extends React.Component {
 
         <div className="row">
           <div className="small-12 columns">
-            <ErrorableSelect required
+            <ErrorableRadioButtonGroup required
```

Comment:
Cool. I peeked in our pre-React application and picked the first radio button example I saw which was relationship. I'll move this back into the parent component.